### ISSUE TITLE
[0.8.2.0 staging-17-n] The possibility to use another's Water Pump and Waste Filter

### ISSUE THREAD

**Issue by SlayksWood** [StrangeLoopGames/EcoIssues#12482]
Yellow - Anna's territory.
Blue - my territory.
![image](https://user-images.githubusercontent.com/45708377/58947475-b00a0d80-8790-11e9-8f4a-db30492a387a.png)
My pipes connecting the Water Pump and Oil Refinery are on the border.
Anna's pipes connecting the Blast Furnace and Waste Filter are on the border too.
If you connect your pipe to another's system, you can use it. I use Anna's Waste Filter, She use my Water Pump. But I have no right to use its territory. As she has no right to my territory.

I think It's wrong behavior.

**Comment by D3nnis3n** [comment]
**This is also used to share water and sewage treatment between players and to manage water and sewage for whole cities by government . Should NOT be fixed or at least having an easy way (authing i do NOT consider an easy way) to still use this!**

**Comment by D3nnis3n** [comment]
Actually players can easily avoid this by not placing pipes near edges of their property. In my opinion - works as intended.

**Comment by Praefator** [comment]
Agreed with Dennis. On my server, we designate a "Hall of Public Works" owned by a government official for our public engineering workshop, as well as hosting the water pumps and later the sewage treatment. Being unable to have a common sewage system and/or water supply would defeat the purpose of these things being a community project, as alluded to in the 8.0 dev streams.
